### ISSUE TITLE
The "Help" dialog contains incorrect heading levels: A11y_Aspire Dashboard_Resources_Heading.

### ISSUE THREAD

**Issue by AnKushSingh05** [dotnet/aspire#10125]
GitHubTags
#A11ySev2; #A11yTCS;#DesktopWeb; #Win32;#.NET Project system for CSharp and VB;#Win32;#WCAG1.3.1;#ScreenReader;#NVDA;#Narrator;#Element:Heading;#Info and Relationships;#A11yMAS;

### Actual Result:
The "Help" dialog contains incorrect heading levels

### Expected Result:
The "Help" heading should use heading level 1, "Keyboard Shortcuts" should use heading level 2, and all other headings should be set as heading level 3.

### Environment Details:
Application Name: Aspire Dashboard
Edition Windows 11 Enterprise
Version 24H2
OS build 26100.4351

### Repro Steps:

1. Open Visual studio.
2. Open a aspire project and run it.
3. A new window will open in the chromium edge, TAB to "Resources" tab item and hit ENTER key.
4. Tab till help button and press enter.
5. The "Help" dialog contains incorrect heading levels.

### User Impact:
Screen reader only users will not be able to understand correct structure of the page and will get confused if incorrect heading levels are provided.

### Attachment:

![Image](https://github.com/user-attachments/assets/0f6db5c3-3c20-4049-8f15-741f876aaf07)
